### PR TITLE
Add docstrings everywhere

### PR DESCRIPTION
**Pull request by lernerbruno**
It adds docstrings everywhere, like the title says :)
I finally did it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+#IDE stuff
+.idea/
```
Comment (Doriaph) [inline .gitignore]: why did you do that?

File: scraper.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
 import requests
 from bs4 import BeautifulSoup
-import quote
```
Comment (Doriaph) [inline scraper.py]: you shouldnt do it, are you crazy?

File: .gitignore (hunk 1 above)
Comment (lernerbruno) [reply .gitignore]: because i dont want to commit .idea files

File: scraper.py (hunk 2 above)
Comment (lernerbruno) [reply scraper.py]: oh yea, Im sorry, Im gonna change that

**Pull request merged**